### PR TITLE
HDFS-12125. Document the missing EC removePolicy command

### PR DESCRIPTION
**Pull request by smengcl**

### CODE REVIEW

File: hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/HDFSErasureCoding.md [hunk 1]
```diff
@@ -218,6 +219,10 @@ Below are the details about each command.
 
      Enable an erasure coding policy.
 
+*  `[-removePolicy -policy <policyName>]`
```
Comment (jojochuang) [inline hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/HDFSErasureCoding.md]: This is a duplicate of line 214.

Comment (smengcl) [reply hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/HDFSErasureCoding.md]: Updated.

**Comment by jojochuang** [comment]
LGTM

**Comment by jojochuang** [comment]
@smengcl you are not yet a Hadoop committer so your PR doesn't trigger precommit builds. I forked your PR as #1264 to facilitate precommit build for you.

**Comment by jojochuang** [comment]
Sorry. I meant PR #1265

**Comment by jojochuang** [comment]
Merged the PR, thanks @smengcl and @wenxinhe !

**Comment by smengcl** [comment]
> @smengcl you are not yet a Hadoop committer so your PR doesn't trigger precommit builds. I forked your PR as #1264 to facilitate precommit build for you.

Thanks for committing!

Does that mean we might want to improve the infra to trigger precommit for new HADOOP/HDFS commits?

**Pull request merged**